main.py

```diff
@@ -16,10 +17,20 @@
 screen = pygame.display.set_mode(size)
 GRAY = (150, 150, 150)
 auto = False
-board = Board(4, 4)
 animate_percentage = 0
 last_direction = 'up'
 
+#Board Logic
+#Allow for command-line arguments of board height and board width (4 being default)
+#Both arguments need to be filled or else something breaks when you try to move
+try: boardw, boardh = int(sys.argv[1]), int(sys.argv[2])
```

Comment:
I would've done an if-else (as that would be a one-liner and it would look nicer), but python throws an error if I try to access `sys.argv`, even if it is for (as an example) `if sys.argv[1] is None`.

Reply:
I guess if you can find another way to do it with an if-else statement then be my guest.

Reply:
on second thought, we'd probably want to handle the case where invalid arguments are given anyway (like `./main.y four four` or something), so we can just leave it as a try/catch for simplicity